sys/shell/commands/sc_gnrc_ipv6_nib.c

```diff
@@ -78,6 +78,15 @@ static void _usage_nib_route(char **argv)
     printf("       %s %s show [iface]\n", argv[0], argv[1]);
 }
 
+static inline gnrc_netif_t *_get_iface(unsigned iface)
+{
+     /* prevent integer overflow can't use pid_is_valid() since it itself would
```

Comment:
this sentence reads quite strange .. do you mind rephrasing it?

Reply:
Done